indico/web/client/js/react/components/PersonLinkField.jsx

```diff
@@ -360,6 +360,12 @@ function PersonLinkField({
               onClose={onClose}
               onSubmit={onSubmit}
               person={persons[selected]}
+              eventId={eventId}
+              otherPersons={
+                selected !== null
+                  ? persons.filter(p => p.email !== persons[selected].email)
```

Comment:
```suggestion
                  ? persons.filter(p => p.id !== persons[selected].id)
```

I think using the person id is more reliable here (assuming we have it)

Reply:
I think in this case it's better to just check the object identity since there are cases where we have neither the id nor email, like when a person is entered manually w/o specifying the email